src/domain/certificates/useCases/getVerificationMap.js

```diff
@@ -0,0 +1,78 @@
+import { BLOCKCHAINS, CERTIFICATE_VERSIONS, STEPS, SUB_STEPS } from '../../../constants';
+import isTestChain from '../../chains/useCases/isTestChain';
```

Comment:
in that case it would still be good to use it as the `chainService` or `domain`.
so either:
`import chainService from '../../chains';`
and then `chainService.isTestChain`

or 
`import domain from '../../'`, and then `domain.chains.isTestChain`;